### PR TITLE
fix: Improve console errors empty state

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
fix: #5923
/claim #5923

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
@mayberryzane @Vadman97

This is ready for review now. Should be exactly like the design.

**Review by Vadman97 — CHANGES_REQUESTED** [review]
the filter state should be updated per the ticket
Screenshot 2023-08-28 at 1 37 30 PM

also, in the design, the text wraps rather than staying as 1 line
image

**Comment by rishi-raj-jain** [comment]
@Vadman97 

Can you help me where I can see that view and reproduce that?

In local, I had to replace one of the random components to be able to get the view of EmptyDevToolsCallout component, and how it's rendered.

**Comment by rishi-raj-jain** [comment]
@Vadman97

Can you help me where I can see that view and reproduce that?

In local, I had to replace one of the random components to be able to get the view of EmptyDevToolsCallout component, and how it's rendered.

**Comment by rishi-raj-jain** [comment]
@Vadman97 @ccschmitz

Can you help me where I can see that view and reproduce that?

In local, I had to replace one of the random components to be able to get the view of EmptyDevToolsCallout component, and how it's rendered.

**Comment by Vadman97** [comment]
@rishi-raj-jain you can look at the [following session here](https://preview.highlight.io/1/sessions/t1j4yjT5cYy0ogLdvFQFJiw2hwQJ?page=1&query=and%7C%7Ccustom_processed%2Cis%2Ctrue%7C%7Ccustom_created_at%2Cbetween_date%2C2023-08-01T06%3A43%3A12.035Z_2023-08-31T06%3A43%3A12.035Z&%7Er_preview=%7B%22action%22%3A%22start%22%2C%22mode%22%3A%22production%22%2C%22region%22%3A%22sjc%22%2C%22variantId%22%3A%2201H8YXFM4JHBYSDCN7Z5BRX2MA%22%2C%22variantData%22%3A%22%7B%5C%22type%5C%22%3A%5C%22branch%5C%22%2C%5C%22branch%5C%22%3A%5C%22fix-info%5C%22%2C%5C%22forkSpecifier%5C%22%3A%5C%22rishi-raj-jain%2Fhighlight%5C%22%2C%5C%22githubOwnerName%5C%22%3A%5C%22highlight%5C%22%2C%5C%22githubRepositoryName%5C%22%3A%5C%22highlight%5C%22%7D%22%7D) 

Screenshot 2023-08-30 at 11 44 34 PM

**Comment by rishi-raj-jain** [comment]
What I see:

Screenshot 2023-08-31 at 9 10 09 PM

**Comment by Vadman97** [comment]
@rishi-raj-jain you may have to view the session in an incognito window (not logged in) to view it publicly

**Comment by rishi-raj-jain** [comment]
@Vadman97 what I see in incognito

Screenshot 2023-09-01 at 9 27 36 PM

**Comment by Vadman97** [comment]
yup, that's correct. now click the terminal icon in the bottom right to open up the console 
image

**Comment by rishi-raj-jain** [comment]
Screenshot 2023-09-02 at 3 27 08 PM

**Comment by rishi-raj-jain** [comment]
@Vadman97 I think for this to be deployed for a preview, needs your authorization

**Comment by rishi-raj-jain** [comment]
@Vadman97 I think for this to be deployed for a preview, needs your authorization

**Comment by rishi-raj-jain** [comment]
@Vadman97 I think for this to be deployed for a preview, needs your authorization

**Comment by rishi-raj-jain** [comment]
@Vadman97 I think for this to be deployed for a preview, needs your authorization

File: frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx [hunk 1]
```diff
@@ -42,13 +50,17 @@ export const EmptyDevToolsCallout = ({
...
-				title={`Couldn't find any recorded ${
-					kind === Tab.Console
-						? 'console messages'
-						: kind === Tab.Network
-						? 'network requests'
-						: kind.toLocaleLowerCase()
-				}.`}
+				title={
+					!requestTypes.length && filter?.length
+						? `Only see one app version?`
```
Comment (Vadman97) [inline frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx:55]:
```suggestion
						? `Can't find what you're looking for?`
```

File: frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx [hunk 2]
```diff
@@ -91,40 +103,39 @@ export const EmptyDevToolsCallout = ({
 				) : filter?.length ? (
 					<>
 						<Text color="n11">
-							Nothing matched filter '{filter}'
-						</Text>
-						<Text color="n11">
-							If you think something's wrong, feel free to reach
-							out to us!
+							Seems like there are no results for your search
+							entry. Try again with another retry!
```
Comment (Vadman97) [inline frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx:107]:
```suggestion
							entry. Try search for something else!
```

File: frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx [hunk 3]
```diff
@@ -34,6 +39,9 @@ export const EmptyDevToolsCallout = ({
 }: Props) => {
 	return (
 		<Box
+			style={{
+				maxWidth: '340px',
+			}}
```
Comment (Vadman97) [inline frontend/src/pages/Player/Toolbar/DevToolsWindowV2/EmptyDevToolsCallout/EmptyDevToolsCallout.tsx:44]: the callout is not centered

Screenshot from 2023-09-05 10-30-39

Screenshot from 2023-09-05 10-29-22

**Review by Vadman97 — COMMENTED** [review]
getting there! some feedback from the preview

**Comment by rishi-raj-jain** [comment]
@Vadman97 This is ready for your review.

**Comment by Vadman97** [comment]
@rishi-raj-jain this is failing the build; please run `yarn reflame-build` and commit the updated files

**Comment by rishi-raj-jain** [comment]
@Vadman97 Done, somehow it wasn't committed.

File: frontend/src/pages/Player/Toolbar/DevToolsWindowV2/ConsolePage/style.css.ts [hunk 4]
```diff
@@ -5,6 +5,9 @@ import { recipe } from '@vanilla-extract/recipes'
 export const consoleBox = style({
 	paddingLeft: 8,
 	height: '100%',
+	display: 'flex',
+	flexDirection: 'row',
+	justifyContent: 'center',
```
Comment (Vadman97) [inline frontend/src/pages/Player/Toolbar/DevToolsWindowV2/ConsolePage/style.css.ts:10]: this still isn't centered

Screenshot from 2023-09-05 22-33-31

image

Comment (Vadman97) [inline frontend/src/pages/Player/Toolbar/DevToolsWindowV2/ConsolePage/style.css.ts:10]: this still isn't centered

Screenshot from 2023-09-05 22-33-31

image

**Review by Vadman97 — CHANGES_REQUESTED** [review]
looks like the box still isn't centered

**Comment by rishi-raj-jain** [comment]
I just wish there was a way for me to test all of this locally!

But anyways, I seem to have applied the css at the wrong place, shall be on the callout now.

Can you deploy and check?

**Comment by rishi-raj-jain** [comment]
I just wish there was a way for me to test all of this locally!

But anyways, I seem to have applied the css at the wrong place, shall be on the callout now.

Can you deploy and check?

**Review by Vadman97 — CHANGES_REQUESTED** [review]
nice! this is centered now, but there is no margin around the callout box as the design has.

as for testing locally, you can easily spin up the stack as per our docs and view the frontend locally. reach out on discord if you have issues with that
https://www.highlight.io/docs/getting-started/self-host/dev-deployment-guide

**Comment by rishi-raj-jain** [comment]
I did do all of the setup, but the issue is I'm not able to load the session link in which the error is visible.

**Comment by rishi-raj-jain** [comment]
> but there is no margin around the callout box as the design has.

What margin are you referring to? I see the design of the callout exactly like the design has.

**Comment by Vadman97** [comment]
some errors now @rishi-raj-jain 

Screenshot 2023-09-11 at 9 05 52 PM

**Review by Vadman97 — APPROVED** [review]
preview is looking good now!
Screenshot 2023-09-11 at 9 07 16 PM

**Comment by rishi-raj-jain** [comment]
That's awesome!

**Pull request merged**